src/components/data_search/DatasetSearchTable.js

```diff
@@ -115,30 +115,39 @@ export const DatasetSearchTable = (props) => {
           data: [
             {
               value: entry[0].study.studyName,
+              truncate: true,
```

Comment:
It would be nicer if there were a default value set to `false` and only specified `true` when necessary.